### PR TITLE
1299: Correctly render YARA search queries

### PR DESCRIPTION
**Pull request by dorpvom**

### CODE REVIEW

File: src/web_interface/filter.py [hunk 1]
```diff
@@ -400,3 +401,11 @@ def elapsed_time(start_time: float) -> int:
 
 def format_duration(duration: float) -> str:
     return str(timedelta(seconds=duration))
+
+
+def render_query_title(query_title: str):
```
Comment (jstucke) [inline src/web_interface/filter.py:406]: Is `query_title` really of type `str` when there are checks whether it is `None` or `dict`?

**Review by jstucke — COMMENTED** [review]
Seems to work well. Should there be a test for the new filter?

File: src/web_interface/filter.py (hunk 1 above)
Comment (dorpvom) [reply src/web_interface/filter.py:406]: Good point. It's either None, a string or a dict

Comment (dorpvom) [reply src/web_interface/filter.py:406]: Though no idea if the None case actually exists, but it was handled before.

**Review by jstucke — APPROVED** [review]

**Pull request merged**